### DISCUSSION TITLE
Good looking standard deign

### DISCUSSION THREAD

**Discussion by spyke7** [community/community#13626] [Other Feature Feedback, Questions, & Ideas]
very well cards. It looks very standard. I like it and I see github forward to release it.

**Comment by jfullstackdev** [comment]
yes 

and 

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
